chroma_core/models/server_profile.py

```diff
@@ -42,6 +42,25 @@ def packages(self):
         default=False, help_text="If True, this profile is presented as the default when adding" "storage servers"
     )
 
+    @property
+    def repo_contents(self):
+        """
+        Convencinece for obtaining the merged contents of the repo file.
+        """
+        contents = ""
+        for repo in self.repolist.all():
```

Comment:
Would this make more sense on the `Repo` model?

Reply:
We still have to iterate over local repo list, but it could be
```python
    return "".join(r.contents for r in  self.repolist.all())
```